sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -475,28 +480,30 @@
      *
      * {@codesnippet com.azure.data.appconfiguration.configurationasyncclient.deleteSetting#ConfigurationSetting}
      *
-     * @param setting The ConfigurationSetting to delete.
+     * @param key The key of the setting to delete.
+     * @param label The label of the setting to delete.
+     * @param eTag The entity tag of the setting to delete.
      * @return The deleted ConfigurationSetting or {@code null} if didn't exist. {@code null} is also returned if the
      * {@code key} is an invalid value or {@link ConfigurationSetting#getETag() etag} is set but does not match the
```

Comment:
This should say `the passed eTag does not match the current eTag`, we aren't pulling the value out of the `ConfigurationSetting` anymore.